### DISCUSSION TITLE
Erase-Install Install Seemingly Stopped After Download

### DISCUSSION THREAD

**Discussion by surchj** [grahampugh/erase-install#487] [Q&A] [answered]
Hello,

I am having an issue where the erase-install.sh script is able to download the installer, but it seems to not be able to actually run it? The logs almost indicate that the script can't find the OS volume to search for the installer. This is for trying minor updates, say from 14.2 -> 14.3.1 Any help would be greatly appreciated. If it helps, we did JUST introduce Microsoft Defender for 365 onto our devices. I put an allow indicator on the .sh file in 365 to see if that would help but haven't gotten anywhere with that clearly. 

Here was the command I was running through Jamf: *Path to script* --sameos --dialog-on-download --update --check-power --cleanup-after-use --no-fs --postinstall-command "Jamf recon"

Thank you!

[erase-install.log](https://github.com/grahampugh/erase-install/files/14314267/erase-install.log)

**Comment by grahampugh** [comment, marked as answer]
You didn't add a --reinstall or --erase flag, so you have only asked it to download.

**Reply by surchj** [reply]
Ah okay that worked! Thank you! I was a little confused about the verbiage/syntax of Upgrade/Update in the update window that the application creates.
